pages/_app.tsx

```diff
@@ -15,6 +15,11 @@ const CustomApp = (appProps: Props): JSX.Element => {
       <Head>
         <title>Forente IT-Foreninger</title>
       </Head>
+      <link
```

Comment:
I think we are supposed to use the _light_ version of this font, at least some places. Should probably include that as well?